nvim/lua/user/plugins.lua

```diff
@@ -86,6 +86,7 @@ return packer.startup(function(use)
     -- run = ":TSUpdate",
   }
 
+  use "alexghergh/nvim-tmux-navigation"
```

Comment:
Fork of https://github.com/christoomey/vim-tmux-navigator that works with pure lua